### PR TITLE
API updates for defiLlama

### PR DESCRIPTION
**Pull request by drugz**
https://github.com/clearpool-finance/clearpool.finance/issues/161

https://github.com/clearpool-finance/clearpool.finance/pull/162

### CODE REVIEW

**Comment by VarinderClearpool** [comment]
@drugz please remember to assign the person you're request review from, also if you could try not to pass code reviews to Vadim, and consider @georgeciubotaru instead, would be appreciated. Thanks

**Comment by drugz** [comment]
> @drugz please remember to assign the person you're request review from, also if you could try not to pass code reviews to Vadim, and consider @georgeciubotaru instead, would be appreciated. Thanks
> 
> Also, please do not move the task into in-review, the reviewer picks-up this task is supposed to pull the task into review. Thanks!

Thank you so much for the clarification! I'll keep that in my mind.

File: src/adaptors/clearpool/index.js [hunk 1]
```diff
@@ -17,7 +20,7 @@ const poolsFunction = async () => {
     };
 
     const dataPools = await utils.getData(
-        'https://clearpool.finance/api/top-pools'
+        'http://app.clearpool.finance/api/pools'
```
Comment (georgeciubotaru) [inline src/adaptors/clearpool/index.js:23]: @drugz Why do we use `http` instead of `https`?

Comment (georgeciubotaru) [inline src/adaptors/clearpool/index.js:23]:
```suggestion
        'https://app.clearpool.finance/api/pools'
```

Comment (drugz) [reply src/adaptors/clearpool/index.js:23]: oh, it is bug... I will fix it, thank you!!

**Review by irondsd — APPROVED** [review]

**Comment by VarinderClearpool** [comment]
@drugz all good? lets go! 🚀

**Comment by drugz** [comment]
https://github.com/DefiLlama/yield-server/pull/479

**Pull request merged**